packages/itwinui-css/src/button/split-button.scss

```diff
@@ -2,46 +2,46 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-button-split-menu {
+@mixin iui-button-split {
   @include iui-button-group;
 
   &:hover {
     background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-6));
   }
 
   > * {
-    &:first-child {
-      > .iui-borderless:hover {
-        background-color: transparent;
-      }
-    }
+    // &:first-child {
+    //   > [data-iui-button-variant='borderless']:hover {
+    //     background-color: transparent;
+    //   }
+    // }
 
     // Sizes
     &:last-child {
       > .iui-button {
         padding: $iui-xs;
       }
 
-      > .iui-small {
+      > [data-iui-button-size='small'] {
         padding: $iui-xxs;
       }
 
-      > .iui-large {
+      > [data-iui-button-size='large'] {
         padding: $iui-s;
       }
     }
 
     // Border between buttons for high-visibility and cta
     &:first-child {
-      > .iui-high-visibility,
-      > .iui-cta {
+      > [data-iui-button-variant='high-visibility'],
+      > [data-iui-button-variant='cta'] {
         border-right-color: rgba(var(--iui-color-foreground-accessory-rgb), var(--iui-opacity-4));
       }
     }
 
     &:last-child {
-      > .iui-high-visibility,
-      > .iui-cta {
+      > [data-iui-button-variant='high-visibility'],
+      > [data-iui-button-variant='cta'] {
```

Comment:
All the overrides in this file look like a good opportunity to use CSS variables.